bin/symfony_requirements

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env php
+<?php
+
+require_once dirname(__FILE__).'/../var/SymfonyRequirements.php';
```

Comment:
Can this symfony requirements binary be an external dependency? Could we bring it in with composer and run it there? Seems rather unrelated to the registry.